### PR TITLE
Packages from splitting cob_environment_perception

### PR DESCRIPTION
**Pull request by ipa-rmb**

### CODE REVIEW

**Comment by ipa-rmb** [comment]
Can we turn on Travis for this repository as well?

**Comment by fmessmer** [comment]
> Can we turn on Travis for this repository as well?

activated --> https://travis-ci.org/ipa320/cob_environment_perception_experimental

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+lib
+build
+docs
+doc
+srv_gen
+msg_gen
+cmake_install.cmake
+*.cfgc
+*/cfg/cpp
+CATKIN_IGNORE
```
Comment (fmessmer) [inline .gitignore]: feel free to use our default: https://github.com/ipa320/care-o-bot/blob/indigo_dev/.gitignore

**Comment by ipa-rmb** [comment]
The current travis error will disappear when cob_environment_perception is merged.

**Comment by ipa-rmb** [comment]
Travis does not complete compilation within the 50 mins. I am merging anyways - maybe travis can handle this in future.

**Comment by ipa-rmb** [comment]
@ipa-fxm Can you please put me in the list of people with write access?

**Comment by fmessmer** [comment]
> @ipa-fxm Can you please put me in the list of people with write access?

done

**Comment by ipa-rmb** [comment]
Thanks!

**Pull request merged**